.github/workflows/pull_request.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       with:
         files: ./cover.out
         flags: unittests
-        fail_ci_if_error: true        
+        fail_ci_if_error: false
```

Comment:
Is that the solution to flaky test coverage ? Will it skip if it fails to upload for example ?

Reply:
yeah well, IMO coverage upload is not something critical enough to make whole CI fail, given how unstable it is